_blogposts/2025-04-05-introducing-unified-operators.mdx

```diff
@@ -0,0 +1,268 @@
+---
+author: rescript-team
+date: "2025-04-05"
+title: "Rethinking Operators: How ReScript Simplifies Arithmetic and Beyond"
+badge: roadmap
+description: |
+    Discover how unified operators in ReScript v12 simplify arithmetic, reduce syntax noise — plus, a glimpse into the future roadmap.
+---
+
+## Introduction
+
+For upcoming ReScript v12, we're upgrading common arithmetic operators to be "Unified Operators"
```

Comment:
In the upcoming ReScript v12, we're upgrading common arithmetic operators to 'Unified Operators".